recipes/edts

```diff
@@ -1,40 +1,30 @@
 (edts
  :fetcher github
- :repo   "tjarvstrand/edts"
+ :repo   "sebastiw/edts"
  :files  ("*.el"
           "COPYING"
           "COPYING.LESSER"
           "Makefile"
           "README.md"
           "start"
+          "start.bat"
+          "rebar.config"
+          ("config"                     "config/*")
 
           ("elisp/edts"                 "elisp/edts/*.el")
           (:exclude                     "elisp/edts/*-test.el")
 
-          ("lib/edts"                   "lib/edts/Makefile")
-          ("lib/edts"                   "lib/edts/rebar")
-          ("lib/edts"                   "lib/edts/rebar.config")
-          ("lib/edts/priv"              "lib/edts/priv/app.config")
+          ("lib/edts/include"           "lib/edts/include/*")
           ("lib/edts/priv"              "lib/edts/priv/dispatch.conf")
           ("lib/edts/src"               "lib/edts/src/*")
 
-          ("plugins/edts_debug"         "plugins/edts_debug/*.el")
-          (:exclude                     "plugins/edts_debug/*-test.el")
-          ("plugins/edts_debug"         "plugins/edts_debug/Makefile")
-          ("plugins/edts_debug"         "plugins/edts_debug/rebar")
-          ("plugins/edts_debug"         "plugins/edts_debug/rebar.config")
-          ("plugins/edts_debug/src"     "plugins/edts_debug/src/*")
+          ("lib/edts_debug"             "lib/edts_debug/*.el")
+          (:exclude                     "lib/edts_debug/*-test.el")
+          ("lib/edts_debug/src"         "lib/edts_debug/src/*")
 
-          ("plugins/edts_dialyzer"      "plugins/edts_dialyzer/*.el")
-          (:exclude                     "plugins/edts_dialyzer/*-test.el")
```

Comment:
I think also future tests in edts_dialyzer should be excluded.
`(:exclude                     "lib/edts_dialyzer/*-test.el")`

Reply:
This directory does not contain any test .el files.

Reply:
No, but if it is included in the future there will have to be a new PR for melpa. My point is that it is easy to forget that and now all files are included. `:exclude` won't do any harm.

Reply:
done